tests/integration/galley/statusupdate/statusupdate_test.go

```diff
@@ -1,133 +0,0 @@
-//  Copyright 2018 Istio Authors
```

Comment:
This should be ported over to istiod since we support status updates there now. @therealmitchconnors, this should mostly be copy/paste except for enabling the analysis feature, right?

Reply:
Mitch already added this to istiod with tests in #22364 - or at least I think this is the same feature?